client/src/js/BrokenExternalLinksReport.js

```diff
@@ -130,6 +143,7 @@
           error(e) {
             if (typeof console !== 'undefined') {
               console.error(e);
+              self.buttonReset();
```

Comment:
Maybe it's only a problem in my local environment, but...
If no report exists yet, then the server returns nothing and the request automatically ends with an error. In this situation, the button is still in the loading state, so I think it's best to return it to its original state